sormas-api/src/main/java/de/symeda/sormas/api/caze/GermanCaseClassificationValidator.java

```diff
@@ -66,7 +67,7 @@ public static boolean isValidGermanCaseClassification(
 	private static boolean hasCoronavirusSymptom(CaseDataDto caseDataDto) {
 		final SymptomsDto symptoms = caseDataDto.getSymptoms();
 		return caseDataDto.getDisease() == Disease.CORONAVIRUS
-			&& symptoms.getSymptomatic()
+			&& symptoms.getSymptomatic() == YesNoUnknown.YES
```

Comment:
Will throw a NullPointerException if "symptomatic" is null. Instead use:
`YesNoUnknown.YES.equals(symptoms.getSymptomatic())`